src/main/java/pixelperfect/Game.java

```diff
@@ -62,14 +68,33 @@ public void simpleInitApp() {
   public void simpleUpdate(float tpf) {
     scheduler.update(tpf);
     spaceship.update(tpf);
-    
+
     if (spaceship.isDead()) {
-      this.stop();
+      // this.stop();
     }
-    
+
     if (spaceship.isVictorious()) {
       System.out.println("Well played, you have completed the game!");
       this.stop();
     }
+
+    debugNetworking();
+  }
+
+  /**
+   * Temporary function for debugging purposes only. Prints updates to the console, when clients
+   * (dis)connect.
+   */
+  public void debugNetworking() {
+    int clients = server.getConnections().size();
+    if (clients > clientCounter) {
+      System.out.println("Clients have connected to the game: "
+          + Integer.toString(clients - clientCounter) + " new clients.");
+      clientCounter = clients;
+    } else if (clients < clientCounter) {
+      System.out.println("Clients have disconnected from the game: "
```

Comment:
Good idea, would you like to refactor this yourself after pulling it in your branch?